src/filters/sse.rs

```diff
@@ -74,7 +75,8 @@ pub trait ServerSentEvent: SseFormat + Sized + Send + 'static {
 impl<T: SseFormat + Send + 'static> ServerSentEvent for T {}
 
 #[allow(missing_debug_implementations)]
-struct SseComment<T>(T);
+/// Comment field (":<comment-text>")
+pub struct SseComment<T>(T);
 
```

Comment:
I'm guessing this needed to become public since it's exposed in the `Stream` implementation of `SseKeepAlive`? I'd prefer to keep it private...

Things that need to be "public" but still private are in the `sealed` mod down below. (They have a `pub` modifier, but since `sealed` isn't public, they're not accessible outside the crate). We could either move this struct into the `sealed` module, or put some `struct SseKeepAliveItem` there instead. Which seems cleaner?